modules/persistent-storage-csi-vsphere-multi-vcenter-support-procedure-install.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// storage/container_storage_interface/persistent-storage-csi-vsphere.adoc
+//
+
+:content-type: PROCEDURE
+[id="persistent-storage-csi-vsphere-multi-vcenter-support-procedure-install_{context}"]
+= Configuring multiple vCenters during installation
+
+To configure multiple vCenters during installation:
+
+* Specify multiple vSphere clusters during installation. See xref:../../installing/installing_vsphere/installation-config-parameters-vsphere.adoc[Installation configuration parameters for vSphere].
```

Comment:
The linked page says "*Currently, only a single vCenter server is supported*", without any TechPreview note about three vCenters.

Is someone else updating the install docs?

Reply:
Yes. I will ask them about getting this fixed and leave this comment open until it's confirmed.

Reply:
This change was [merged](https://docs.openshift.com/container-platform/4.17/installing/installing_vsphere/installation-config-parameters-vsphere.html#installation-configuration-parameters-additional-vsphere_installation-config-parameters-vsphere) yesterday.

Reply:
Resolved.